src/main/java/io/swagger/api/impl/InvoicesApiServiceImpl.java

```diff
@@ -172,6 +167,29 @@ public Response getInvoiceState(UUID invoiceId, SecurityContext securityContext)
         return resp;
     }
     @Override
+    public Response getInvoiceTransferState(UUID invoiceId, SecurityContext securityContext) throws NotFoundException {
+        Response resp;
+        Error err = new Error();
+        err.setMessage("success");
+        try {
+            List<AddressValuePair> transfers = Bitcoin.getInstance().getInvoiceTransfers(invoiceId);
```

Comment:
It shall not be checked here, if there do exist transfers for an invoice. Preferably `Bitcoin.getInvoiceTransferState()`shall throw an exception if there are no transfers for the invoice.